src/core/include/ngraph/util.hpp

```diff
@@ -76,6 +76,8 @@ std::string trim(const std::string& s);
 NGRAPH_API
 std::vector<std::string> split(const std::string& s, char delimiter, bool trim = false);
 
+bool check_all_digits(const std::string& value);
```

Comment:
Please don't extend legacy API. If we need some helper please extend new public or developer API

Reply:
Moved to check_all_digits() to openvino/utils/common_utils.hpp